.gitignore

```
@@ -1 +1,3 @@
-.history
\ No newline at end of file
+_site
```

1. I am using a VS code plugin which tracks every file change in a folder called `.history`. This is a plan B in cases something goes wrong with git and my changes (untracked files) are gone or something else fails. So I would like to have it. 

2. Moreover, I was thinking about using this plugin with students, as well because it automatically tracks file changes but we can discuss this later

```suggestion
.history
_site
```

---

Sounds good. I didn't look at this change properly, sorry it got deleted